### ISSUE TITLE
We need to sort out axis labeling soon

### ISSUE THREAD

**Issue by matthew-brett** [nipy/nibabel#412]
I've just merged #255 - which assumes that the spatial axes are the first three.

This is true of Nifti and Analyze, but not - say - of MINC.

See discussion at https://github.com/nipy/nibabel/pull/255#issuecomment-60805478 and https://github.com/nipy/nibabel/wiki/BIAP6#general-solution-associating-axes-and-labels

Before the next release we should work out how to specifically identify axes, either by forcing MINC into the NIFTI convention, or by specific labeling.

I'm leaning towards depending on [datarray](https://github.com/fperez/datarray) for this.

**Comment by jona-sassenhagen** [comment]
>I'm leaning towards depending on datarray for this.

This means you have considered, and decided against, https://github.com/pydata/xarray ?

**Comment by matthew-brett** [comment]
Yes, at least for now.  xarray depends on Pandas, and nibabel is a python-only package at the moment.
